airflow-core/src/airflow/ui/public/i18n/locales/zh-TW/components.json

```
@@ -86,6 +86,11 @@
     "taskGroup": "任務群組"
   },
   "limitedList": "+ 其他 {{count}} 項",
+  "limitedList.allItems": "所有 {{count}} 項目：",
+  "limitedList.clickToInteract": "點擊標籤以篩選 DAGs",
+  "limitedList.clickToOpenFull": "點擊 \"+{{count}} 更多\" 以開啟完整檢視",
+  "limitedList.copyPasteText": "你可以複製並貼上上方文字",
+  "limitedList.showingItems": "顯示 {{count}} 項目",
```

```suggestion
  "limitedList.showingItems": "顯示 {{count}} 個項目",
```

---

@Lee-W  I think you mean

```suggestion
  "limitedList.showingItems": "顯示 {{count}} 個項目",
```

right?

---

you're right hahahaha